df.history.xml

```diff
@@ -343,22 +343,22 @@
             <stl-string name="name_adjective"/>
             <int32_t name='race' ref-target='creature_raw'/>
             <int16_t name='caste' ref-target='caste_raw' aux-value='$$.race'/>
-            <stl-vector name='unk_a0' type-name='int32_t'/>
+            <stl-vector name='body_transformation_effects' type-name='int32_t' ref-target='creature_interaction_effect_body_transformationst'/>
             <stl-vector name='unk_b0' type-name='int32_t'/>
-            <stl-vector name='unk_c0' type-name='int32_t'/>
+            <stl-vector name='body_mat_interaction_effects' type-name='int32_t' ref-target='creature_interaction_effect_body_mat_interactionst'/>
```

Comment:
I would lean towards making the ref-targets `creature_interaction_effect`, i.e. the base type, since that's the type that has an `instance-vector` and has a static `find()` method defined in C++. (In Lua, `df.creature_interaction_effect_body_mat_interactionst.find()` *is* defined, although it can return other subtypes of `creature_interaction_effect`, so it's really just an alias for `df.creature_interaction_effect.find()`.)

Reply:
Got it, done.